ait/_windows.py

```diff
@@ -307,7 +307,10 @@
 
 
 def _key_to_vk(key):
-    key = key.strip(' ').upper()
+    if set(key) == {' '}: # only spaces?
+        key = ' '
+    else:
+        key = key.strip(' ').upper()
```

Comment:
I don't think we should worry about `_key_to_vk('        ')`.

```suggestion
    if key != ' ':
        key = key.strip(' ').upper()
```

Reply:
agreed - see my latest commit below